communities/astropy.md

```diff
@@ -32,7 +32,7 @@ core package for Astronomy in Python. Astropy also fosters an ecosystem of
 interoperable astronomy packages. Please remember to acknowledge and cite the use of any {{ page.community }}
 packages that you use.
 
-Astropy currently has {{ total_packages }} packages that have been accepted by pyOpenSci and also become affiliated Astropy packages through our [partnership](partners.html).
+Astropy currently has {{ total_packages }} packages that have been accepted by pyOpenSci and also become affiliated Astropy packages through our [partnership](/partners.html).
```

Comment:
This fix is copied from L12 above which is not a broken link in production but does not seem to want to render in PR preview: `{"message":"Route Not Found"}` 🤷‍♀️

Reply:
@pllim is it broken in circle CI? we do have some issues with relative links and images not rendering in our CI. I haven't had time to try to troubleshoot it but i think it relates to not having {{ base.url }} configured properly.

Reply:
Configuring for `base.url` is beyond me because the URL for each PR review is different. I think the preview bot needs to be smart enough to set this dynamically. Over as Astropy, we use RTD preview so we never have to worry about setting that manually.

Reply:
All I am pointing out is that the fix in this line cannot really be proven to work until after merge.

Reply:
i hear you. i wasn't asking you to fix anything. we've just had ongoing issues with our preview not always working. So the best thing to do it to build locally. 

when i build this pr locally, it all looks fine to me - the links work etc . Let's merge this. Then i opened 3 issues in the pyosmeta repo that highlight the issues with the data that we are seeing. 

when we merge we can see if the astropy website works again as well. 

what's hardest with our data generation here is that people always do slightly different things in issues. so here, the maintainers aren't listed with commas for stingray. we could fix that in the issue but i think we should also account for the other items that you fixed here over at pyosmeta.